From 8f27657605754b6ef8ad4aff55c1cd98deff152d Mon Sep 17 00:00:00 2001
From: Lukas Czerner <lczerner@redhat.com>
Date: Wed, 29 Oct 2014 16:34:20 -0400
Subject: [fs] ext4: fix comments about get_blocks

Message-id: <1414600485-1929-31-git-send-email-lczerner@redhat.com>
Patchwork-id: 98540
O-Subject: [RHEL 7.1 PATCH v2 30/55] ext4: fix comments about get_blocks
Bugzilla: 1156096
RH-Acked-by: Eric Sandeen <sandeen@redhat.com>

From: Seunghun Lee <waydi1@gmail.com>

BZ 1156096
https://bugzilla.redhat.com/show_bug.cgi?id=1156096

BREW 8169722
https://brewweb.devel.redhat.com/taskinfo?taskID=8169722

Upstream commit d91bd2c1d78d8d22f9f721aae84650a08239b509

 get_blocks is renamed to get_block.

 Signed-off-by: Seunghun Lee <waydi1@gmail.com>
 Signed-off-by: Theodore Ts'o <tytso@mit.edu>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/ext4/inode.c b/fs/ext4/inode.c
index 344e909..8cfe2cf 100644
--- a/fs/ext4/inode.c
+++ b/fs/ext4/inode.c
@@ -589,7 +589,7 @@ found:
  /*
   * New blocks allocate and/or writing to unwritten extent
   * will possibly result in updating i_data, so we take
-  * the write lock of i_data_sem, and call get_blocks()
+  * the write lock of i_data_sem, and call get_block()
   * with create == 1 flag.
   */
  down_write(&EXT4_I(inode)->i_data_sem);
@@ -1532,7 +1532,7 @@ out_unlock:
 }
 
 /*
- * This is a special get_blocks_t callback which is used by
+ * This is a special get_block_t callback which is used by
  * ext4_da_write_begin().  It will either return mapped block or
  * reserve space for a single block.
  *
-- 
1.7.1